gnucash/gtkbuilder/dialog-preferences.glade

```diff
@@ -3280,6 +3280,7 @@ many months before the current month:</property>
                     <property name="visible">True</property>
                     <property name="can_focus">True</property>
                     <property name="tooltip_text" translatable="yes">To retrieve online quotes from Alphavantage, this key needs to be set. A key can be retrieved from the Alpha Vantage website.</property>
+                    <property name="width_chars">20</property>
```

Comment:
On my system this is still not sufficient if the key happens to consist of only very wide glyphs (try 16x 'W'). Instead you could try to make the field expand (under common widget properties in glade).

Reply:
I assumed these were hexadecimal characters for the key.